Lib/test/test_sqlite3/test_userfunctions.py

```diff
@@ -223,6 +224,14 @@ def test_func_error_on_create(self):
         with self.assertRaises(sqlite.OperationalError):
             self.con.create_function("bla", -100, lambda x: 2*x)
 
+    def test_func_too_many_args(self):
+        category = sqlite.SQLITE_LIMIT_FUNCTION_ARG
+        msg = "too many arguments on function"
+        with cx_limit(self.con, category=category, limit=1):
+            with self.assertRaisesRegex(sqlite.OperationalError, msg):
+                self.con.create_function("addint", 2, lambda x, y: x+y)
+                self.con.execute("select addint(1, 2)");
```

Comment:
It is not clear what of calls is failed: `create_function()` or `execute()`?

Reply:
True. I'll move the function creation out of the assert.